apps/checkout-app/.env.template

```diff
@@ -1,5 +1,7 @@
 VERCEL_URL=http://localhost:3000
 NEXT_PUBLIC_VERCEL_URL=${VERCEL_URL}
 NEXT_PUBLIC_API_URL=https://master.staging.saleor.cloud/graphql/
+NEXT_PUBLIC_SALEOR_URL=https://master.staging.saleor.cloud/
```

Comment:
We need this for fetching jwks public key from `NEXT_PUBLIC_SALEOR_URL/.well-known/jwks.json`

Reply:
You can get this email from the access token instead. Let's try to write the app as multi-tenat as we can at the moment  (minimize hardcoding values via env variables) so it will be easier to migrate in the future.